# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/st/09-config.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- st-0.9.2/config.def.h	2022-10-04 19:41:26.000000000 +0200
+++ st-0.9.2/config.def.h	2024-04-14 19:41:48.537682406 +0200
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "DejaVu Sans Mono:size=14";
 static int borderpx = 2;
 
 /*
@@ -32,9 +32,9 @@
 /*
  * word delimiter string
  *
- * More advanced example: L" `'\"()[]{}"
+ * More advanced example: L" `'\"()[]{}@"
  */
-wchar_t *worddelimiters = L" ";
+wchar_t *worddelimiters = L" `'\"()[]{}@";
 
 /* selection timeouts (in milliseconds) */
 static unsigned int doubleclicktimeout = 300;
